access/api.go

```diff
@@ -314,9 +314,10 @@ type CompatibleRange struct {
 
 // NodeVersionInfo contains information about node, such as semver, commit, sporkID, protocolVersion, etc
 type NodeVersionInfo struct {
-	Semver               string
-	Commit               string
-	SporkId              flow.Identifier
+	Semver  string
+	Commit  string
+	SporkId flow.Identifier
+	// TODO: should we replicate protocol state version here, or just remove this field?
 	ProtocolVersion      uint64
```

Comment:
Should we replicate protocol state version here, or omit this field?

Reply:
At the moment, we are using the protocol state synonymously with the version of the protocol, so it might be useful to report it here.

Reply:
I think we should keep it and have it use the protocol state version. We should also include all other important versions here as well (e.g. execution engine version)

Reply:
regarding the already-existing `ProtocolVersion`, was just hoping to confirm: a version change from 37 (old versioning scheme) to 2 (new scheme) would be fine?